### DISCUSSION TITLE
Welcome to MultiRPC Discussions!

### DISCUSSION THREAD

**Discussion by Azyyyyyy** [FluxpointDev/MultiRPC#14] [Old]
## 👋 Welcome!
  We’re going to be using Discussions as a place for people to ask about upcoming features or about features that have been requested for MultiRPC!

If you want to request anything then fire away by creating a new Discussion and we'll get to you shortly about the request.

**Comment by xXBuilderBXx** [comment]
Redoing MultiRPC
